generator/src/main/java/com/scottlogic/deg/generator/App.java

```diff
@@ -1,14 +1,13 @@
 package com.scottlogic.deg.generator;
 
+import com.scottlogic.deg.generator.CommandLine.GenerateCommandLine;
 import picocli.CommandLine;
 
-import java.util.stream.Collectors;
-
 @CommandLine.Command(name = "dg")
-public class App implements Runnable {
+public class App {
     private static final CommandLine picoCliCommandLine = new CommandLine(new App())
-        .addSubcommand("generate", new Generate())
-        .addSubcommand("generateTestCases", new GenerateTestCases())
+        .addSubcommand("generate", new GenerateCommandLine())
+//         TODO - add other commands
```

Comment:
Can you remove this comment, and create a ticket to cover phase 2 of IoC adoption